src/SignalRServiceExtension/TriggerBindings/SignalRTriggerBindingProvider.cs

```diff
@@ -99,41 +101,21 @@ internal SignalRTriggerAttribute GetParameterResolvedAttribute(SignalRTriggerAtt
                 ? parameterNamesFromAttribute
                 : parameterNames;
 
-            var resolvedConnectionString = GetResolvedConnectionString(
-                typeof(SignalRTriggerAttribute).GetProperty(nameof(attribute.ConnectionStringSetting)),
-                attribute.ConnectionStringSetting);
-
-            return new SignalRTriggerAttribute(hubName, category, @event, parameterNames) {ConnectionStringSetting = resolvedConnectionString};
+            return new SignalRTriggerAttribute(hubName, category, @event, parameterNames) { ConnectionStringSetting = attribute.ConnectionStringSetting };
```

Comment:
Any reason to move the resolved connection string to a separate parameter of `SignalRTriggerBinding` instead of put it into `SignalRTriggerAttribute`

Reply:
In the multiple-endpoint scenario, there are multiple connection strings and we need to resolve them with customized method in SDK. So resolving only one connection string outside SDK will be meaningless. The connection string parameter of  `SignalRTriggerBinding` will be replaced by connection string array or access key array in the future.